atlasdb-api/src/main/java/com/palantir/atlasdb/keyvalue/api/ColumnSelection.java

```diff
@@ -44,12 +43,12 @@ private ColumnSelection(SortedSet<byte[]> selectedColumns) {
 
     public static ColumnSelection valueOf(String serialized) {
         Set<byte[]> columns = Sets.newTreeSet(UnsignedBytes.lexicographicalComparator());
-        for (String strColumn : serialized.split("\\s*,\\s*")) {
-            strColumn = strColumn.trim();
-            if (strColumn.equals("")) {
+        for (String columnString : serialized.split("\\s*,\\s*")) {
+            String trimmedColumnString = columnString.trim();
```

Comment:
When you have to do some pre-processing on each element (here, trimming `columnString`), it would be nice to have the rest of the `for` loop in a separate method. This obeys (or is closer to obeying) the One Thing Rule, and avoids the risk of a careless programmer using `columnString` elsewhere in the loop body at some point in the future.